relayer/relays/parachain/beefy-listener.go

```diff
@@ -775,7 +775,12 @@ func fetchBundleProof(
 	var rawProof RawMerkleProof
 	var bundleProof BundleProof
 
-	err := api.Client.Call(&proofHex, "basicOutboundChannel_getMerkleProof", digestItemHash, bundleIndex)
+	digestItemHex, err := types.EncodeToHexString(digestItemHash)
```

Comment:
This parameter is incorrectly named: `digestItemHash`. Should be `commitmentHash`.

```suggestion
	commitmentHashHex, err := types.EncodeToHexString(commitmentHash)
```